runners/wikiVsCode/templates/createFile.asciidoc

```diff
@@ -0,0 +1,23 @@
+== Create the file <%= fileName; %>
+
+
+=== Creating <%= fileName; %> in the VSCode Editor
+
+Create <%= fileName; %> in the VSCode editor<% if(contentFile) { %> and insert some data from <%= contentFile; %> into it.
+
+First, you want to open <%= contentFile; %> to get the data we want to put into <%= fileName; %>.
+Go to the file context menu in the top left corner right next to the VS Code symbol and choose *Open File...* or use the keyboard shortcut ctrl+o. 
```

Comment:
*VSCode

Reply:
I think we should use either 'VSCode' or 'VS Code'

Reply:
I will change it to VS Code.